content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -34,6 +34,21 @@ For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overvi
 
 For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.
 
+### LWM2M 1.1 support
+
+We are now supporting the following LWM2M 1.1 features:
+
+- LWM2M Send
+- Composite Operations (read, write, observe, cancel observation)
+- New LWM2M serialization formats: CBOR, SenML JSON, SenML CBOR
+- New LWM2M data formats: Core Link, Unsigned Integers
+
+For more information on our 1.1 support, see our [LwM2M protocol integration guide](http://localhost:58835/guides/protocol-integration/lwm2m/).
```

Comment:
The links point to your local environment.

Reply:
Whoopsie!